content/CORE/Storage/Pools/StorageEncryption.md

```diff
@@ -50,136 +49,273 @@ These algorithms provide authenticated encryption with block ciphers.
+
+There are two ways to manage the encryption credentials: with key files or passphrases.
 
 {{< tabs "Encryption Credentials" >}}
 {{< tab "Key Files" >}}
-#### Key Files
 
 Creating a new encrypted pool automatically generates a new key file and prompts you to download it.
-**Always back up the key file to a safe and secure location.**
+
+{{< hint danger>}}
+*Always back up the key file to a safe and secure location.*
+{{< /hint >}}
 
 ![EncryptionKeyBackupWarning](/images/CORE/12.0/EncryptionKeyBackupWarning.png "Encryption Backup Warning")
 
-Manually download a copy of the pool's inherited and non-inherited encrypted dataset keyfiles by opening the pool <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and selecting *Export Dataset Keys*. Enter the root password and click the *CONTINUE* button.
+To manually download a copy of the pool's inherited and non-inherited encrypted dataset key files, open the pool <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click on **Export Dataset Keys**. Enter the root password and click the **CONTINUE** button.
 
 ![StoragePoolsEncryptionActionsExportKeys](/images/CORE/12.0/StoragePoolsEncryptionActionsExportKeys.png "Exporting Key Files")
 
 {{< hint info >}}
-To manually download a back up of a single dataset's keyfile, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and select *Export Key*. Enter the root password and click the *CONTINUE* button. Click the *DOWNLOAD KEY* button.
+To manually download a backup a single dataset's key file, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and click on **Export Key**. 
+Next type the root password and click the **CONTINUE** button. Click the **DOWNLOAD KEY** button.
 {{< /hint >}}
 
-To change the key, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and *Encryption Options*.
+To change the key, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and **Encryption Options**.
 
 ![StoragePoolsEncryptedDataset](/images/CORE/12.0/StoragePoolsEncryptedDataset.png "Dataset Options: Encryption Options")
 
-Enter your custom key or click *Generate Key*.
+Type your custom key or click **Generate Key**.
 
 ![StoragePoolsEncryptedDatasetOptions](/images/CORE/12.0/StoragePoolsEncryptedDatasetOptions.png "Editing Encryption Options")
+
 {{< /tab >}}
 {{< tab "Passphrases" >}}
-#### Passphrases
 
-To use a passphrase instead of a keyfile, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and *Encryption Options*.
-Change the *Encryption Type* from *Key* to *Passphrase*.
+To use a passphrase instead of a keyfile, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> icon to dispaly the **Dataset Actions** menu and then click **Encryption Options**.
```

Comment:
typo
```suggestion
To use a passphrase instead of a keyfile, click the dataset <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> icon to display the **Dataset Actions** menu and then click **Encryption Options**.
```